translate/storage/test_po.py

```diff
@@ -537,64 +511,65 @@ def test_makeobsolete_plural(self):
 #~ msgstr[1] "Koeie"
 '''
         pofile = self.poparse(posource)
-        print(pofile)
         unit = pofile.units[0]
         assert not unit.isobsolete()
         unit.makeobsolete()
         assert unit.isobsolete()
-        print(pofile)
         assert str(unit) == poexpected
 
     def test_makeobsolete_msgctxt(self):
         """Tests making a unit with msgctxt obsolete"""
-        posource = '#: test.c\nmsgctxt "Context"\nmsgid "test"\nmsgstr "rest"\n'
-        poexpected = '#~ msgctxt "Context"\n#~ msgid "test"\n#~ msgstr "rest"\n'
+        posource = ('#: test.c\n'
+                    'msgctxt "Context"\n'
+                    'msgid "test"\nmsgstr "rest"\n')
+        poexpected = ('#~ msgctxt "Context"\n'
+                      '#~ msgid "test"\n#~ msgstr "rest"\n')
         pofile = self.poparse(posource)
-        print(pofile)
         unit = pofile.units[0]
         assert not unit.isobsolete()
         assert unit.istranslatable()
         unit.makeobsolete()
         assert unit.isobsolete()
         assert not unit.istranslatable()
-        print(pofile)
         assert str(unit) == poexpected
 
     def test_makeobsolete_msgidcomments(self):
         """Tests making a unit with msgidcomments obsolete"""
-        posource = '#: first.c\nmsgid ""\n"_: first.c\\n"\n"test"\nmsgstr "rest"\n\n#: second.c\nmsgid ""\n"_: second.c\\n"\n"test"\nmsgstr "rest"'
+        posource = ('#: first.c\nmsgid ""\n"_: first.c\\n"\n"test"\n'
+                    'msgstr "rest"\n\n'
+                    '#: second.c\nmsgid ""\n"_: second.c\\n"\n"test"\n'
+                    'msgstr "rest"')
         poexpected = '#~ msgid ""\n#~ "_: first.c\\n"\n#~ "test"\n#~ msgstr "rest"\n'
-        print("Source:\n%s" % posource)
-        print("Expected:\n%s" % poexpected)
         pofile = self.poparse(posource)
         unit = pofile.units[0]
         assert not unit.isobsolete()
         unit.makeobsolete()
         assert unit.isobsolete()
-        print("Result:\n%s" % pofile)
         assert str(unit) == poexpected
 
     def test_multiline_obsolete(self):
         """Tests for correct output of mulitline obsolete messages"""
-        posource = '#~ msgid ""\n#~ "Old thing\\n"\n#~ "Second old thing"\n#~ msgstr ""\n#~ "Ou ding\\n"\n#~ "Tweede ou ding"\n'
+        posource = ('#~ msgid ""\n'
+                    '#~ "Old thing\\n"\n'
+                    '#~ "Second old thing"\n'
+                    '#~ msgstr ""\n'
+                    '#~ "Ou ding\\n"\n'
+                    '#~ "Tweede ou ding"\n')
```

Comment:
In other test files the source is put in a triple quoted string, which allows to just copy-paste the real sample because we don't need to escape. Also it is easier to read. I suggest you to rework all these `posource` to use triple quoted strings, like:

```python
        posource = """#~ msgid ""
#~ "Old thing\n"
#~ "Second old thing"
#~ msgstr ""
#~ "Ou ding\n"
#~ "Tweede ou ding"
"""
```